DeviceManager/LoggerHandler.py

```diff
@@ -0,0 +1,67 @@
+from flask import Blueprint, request, jsonify, make_response
+
+from DeviceManager.conf import CONFIG
+from DeviceManager.Logger import Log
+
+from DeviceManager.app import app
+
+from DeviceManager.SerializationModels import parse_payload, load_attrs
+from DeviceManager.SerializationModels import log_schema
+from DeviceManager.utils import format_response, HTTPRequestError, get_pagination
+
+logger = Blueprint('log', __name__)
+LOG = Log()
+
+class LoggerHandler:
+    
+    def _init_(self):
+        pass
+    
+    @staticmethod
+    def update_log_level(level):
+        """
+        Update the log level of device manager.
+
+        :param level: Receive a string containing the new log level.
+        :raises HTTPRequestError: If no authorization token was provided (no
+        tenant was informed) or the request body contains some error.
+        """
+
+        LOG.update_log_level(level.upper())
+    
+    @staticmethod
+    def get_log_level():
+        """
+        Fetches the log level configured.
+
+        :return A JSON containing the log level.
+        :rtype JSON
+        """
+
+        result = {
+            'level': LOG.get_log_level()
+        }
+
+        return result
+
+
+@logger.route('/log', methods=['PUT'])
+def flask_update_log_level():
+    try:
+        _, json_payload = parse_payload(request, log_schema)
```

Comment:
for the sake of curiosity,  if the HTTP request contains an invalid JSON it throws an HTTPRequestError?

Reply:
Yes, the method called parse_payload validate the JSON format of request and also the fields passed like argument in schema, returning a HTTPRequestError.

Reply:
Nice